File: core/src/main/java/com/nfthub/core/repository/MagazineRepository.java
package com.nfthub.core.repository;

import com.nfthub.core.entity.Magazine;
import com.nfthub.core.repository.support.MagazineRepositorySupport;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface MagazineRepository extends JpaRepository<Magazine, Long>, MagazineRepositorySupport {
}

File: src/main/kotlin/com/nfthub/api/repository/MagazineRepository.kt
package com.nfthub.api.repository

import com.nfthub.api.entity.Magazine
import com.nfthub.api.entity.MagazineImage
import com.nfthub.api.entity.MagazineTag
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface MagazineRepository : JpaRepository<Magazine, Long> {
}

@Repository
interface MagazineTagRepository : JpaRepository<MagazineTag, Long>

@Repository
interface MagazineImageRepository : JpaRepository<MagazineImage, Long>